### ISSUE TITLE
Publish Trezor Suite Lite on F-droid

### ISSUE THREAD

**Issue by yagarea** [trezor/trezor-suite#10074]
It would be nice to be able to download Trezor Suite Lite from [F-Droid store](https://f-droid.org/). This would make Trezor Suite Lite available on ungoogled phones.

**Comment by mnuky** [comment]
@matejkriz is this issue part of this issue? https://github.com/trezor/trezor-suite/issues/10789

**Comment by matejkriz** [comment]
No, it's not part of it. https://github.com/trezor/trezor-suite/issues/10789 is also solution for ungoogled phones, but publishing to F-Droid store can be better UX for some users. But when we publish the APK on github, this will be less of a priority.

**Comment by yagarea** [comment]
> is also solution for ungoogled phones, but publishing to F-Droid store can be better UX for some users. But when we publish the APK on github, this will be less of a priority.

Managing updates manually from github is not a ideal. If important security bug will be discovered F-droid would provide much quicker distribution of critical updates for ungoogled users.
